app/(saas)/hackathons/[hackathonSlug]/layout.tsx

```
@@ -6,6 +6,7 @@ import { HackathonEvents } from "@/app/(saas)/hackathons/[hackathonSlug]/_featur
 import { HackathonSummary } from "@/app/(saas)/hackathons/[hackathonSlug]/_features/hackathon-summary/hackathon-summary";
 import { RegisterHackathon } from "@/app/(saas)/hackathons/_features/register-hackathon/register-hackathon";
 
+import { Button } from "@/design-system/atoms/button/variants/button-default";
```

ESLint

[error] 'Button' is defined but never used.

(unused-imports/no-unused-imports)